java/java-ranger-regression/schedule_eqchk/Main.java

```diff
@@ -0,0 +1,19 @@
+
+public class Main {
+    Process[] runSchedule2(int in0, int in1, int in2) {
+        Schedule2 t = new Schedule2();
+        t.mainProcess(in0, in1, in2);
+        return Schedule2.block_queue;
+    }
+
+
+    public static void main(String[] args) {
+        TestSchedule2 t = new TestSchedule2();
```

Comment:
From my point of view, this code example seems to mix infrastructure code required by SPF based tools for analysis with the code under verification. Could you separate this? I my opinion, the verification task should only contain code that is part of the system under verification.